src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${this.name}: x=${
+        this.coords.x}, y=${
+        this.coords.y}`;
```

Comment:
```suggestion
      return `${this.name}: ` 
      + `x=${this.coords.x}, ` 
      + `y=${this.coords.y}`;
```
Так трохи краще читається